web/ui/static/js/prom_console.js

```diff
@@ -523,6 +523,16 @@ PromConsole.Graph.prototype._clearGraph = function() {
 
 PromConsole.Graph.prototype._xhrs = [];
 
+// Enable a user to build a query in the context of this graph
+// e.g. relevant annotations
```

Comment:
This part of comment doesn't make sense in terms of the code, I'd suggest removing it.

Comments should end in full stops.